src/app/models/login.ts

```diff
@@ -0,0 +1,18 @@
+export class LoginResponse {
+
+  constructor(id?: number, statusCode?: number) {
+    this._id = id;
+    this._statusCode = statusCode;
+  }
+
+  private _id: number;
+  private _statusCode: number;
```

Comment:
please put the fields before the constructor. this is my styleguide request, fields first, then functions starting with the constructor.

Reply:
Fixed, thank you!